wallets/react-wallet-v2/src/utils/PolkadotWalletUtil.ts

```diff
@@ -11,6 +12,14 @@ let address2: string
 /**
  * Utilities
  */
+export function getPolkadotWallets(address: string) {
+  Object.entries(polkadotWallets)
```

Comment:
This expression isn't assigned/seems to duplicate the line below

Reply:
Uh. good catch!